### ISSUE TITLE
Sersic drawImage fails for some values of n just under 0.5

### ISSUE THREAD

**Issue by taranu** [GalSim-developers/GalSim#1041]
```
galsim.Sersic(n=0.499999999999999, half_light_radius=1).drawImage()
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-15-2f9d139261c0> in <module>()
----> 1 galsim.Sersic(n=0.499999999999999, half_light_radius=1).drawImage()

~/.local/lib/python3.6/site-packages/galsim/gsobject.py in drawImage(self, image, nx, ny, bounds, scale, wcs, dtype, method, area, exptime, gain, add_to_image, use_true_center, offset, n_photons, rng, max_extra_noise, poisson_flux, sensor, surface_ops, n_subsample, maxN, save_photons, setup_only)
   1632 
   1633         # Make sure image is setup correctly
-> 1634         image = prof._setup_image(image, nx, ny, bounds, add_to_image, dtype)
   1635         image.wcs = wcs
   1636 

~/.local/lib/python3.6/site-packages/galsim/gsobject.py in _setup_image(self, image, nx, ny, bounds, add_to_image, dtype, odd)
   1068                 image = Image(nx, ny, dtype=dtype)
   1069             else:
-> 1070                 N = self.getGoodImageSize(1.0)
   1071                 if odd: N += 1
   1072                 image = Image(N, N, dtype=dtype)

~/.local/lib/python3.6/site-packages/galsim/gsobject.py in getGoodImageSize(self, pixel_scale)
   1768         """
   1769         # Start with a good size from stepk and the pixel scale
-> 1770         Nd = 2. * math.pi / (pixel_scale * self.stepk)
   1771 
   1772         # Make it an integer

~/.local/lib/python3.6/site-packages/galsim/gsobject.py in stepk(self)
    302     def stepk(self):
    303         "The sampling in k space necessary to avoid folding of image in x space."
--> 304         return self._stepk
    305 
    306     @property

~/.local/lib/python3.6/site-packages/galsim/convolve.py in _stepk(self)
    295         # Assume R_final^2 = Sum(R_i^2)
    296         # So 1/stepk^2 = 1/Sum(1/stepk_i^2)
--> 297         inv_stepksq_list = [obj.stepk**(-2) for obj in self.obj_list]
    298         return np.sum(inv_stepksq_list)**(-0.5)
    299 

~/.local/lib/python3.6/site-packages/galsim/convolve.py in <listcomp>(.0)
    295         # Assume R_final^2 = Sum(R_i^2)
    296         # So 1/stepk^2 = 1/Sum(1/stepk_i^2)
--> 297         inv_stepksq_list = [obj.stepk**(-2) for obj in self.obj_list]
    298         return np.sum(inv_stepksq_list)**(-0.5)
    299 

~/.local/lib/python3.6/site-packages/galsim/gsobject.py in stepk(self)
    302     def stepk(self):
    303         "The sampling in k space necessary to avoid folding of image in x space."
--> 304         return self._stepk
    305 
    306     @property

~/.local/lib/python3.6/site-packages/galsim/utilities.py in __get__(self, obj, cls)
   1393         if obj is None:
   1394             return self
-> 1395         value = self.fget(obj)
   1396         setattr(obj, self.func_name, value)
   1397         return value

~/.local/lib/python3.6/site-packages/galsim/transform.py in _stepk(self)
    421     def _stepk(self):
    422         self._major_minor()
--> 423         stepk = self._original.stepk / self._major
    424         # If we have a shift, we need to further modify stepk
    425         #     stepk = Pi/R

~/.local/lib/python3.6/site-packages/galsim/gsobject.py in stepk(self)
    302     def stepk(self):
    303         "The sampling in k space necessary to avoid folding of image in x space."
--> 304         return self._stepk
    305 
    306     @property

~/.local/lib/python3.6/site-packages/galsim/sersic.py in _stepk(self)
    324     @property
    325     def _stepk(self):
--> 326         return self._sbp.stepK()
    327 
    328     @property

RuntimeError: Solve error: Too many iterations in bracketUpper()
```

This is on a freshly-installed master, but the same thing also happens on 2.0 with Python 3.7.

Other values close to 0.5 also fail, not just the one I picked. This might be avoided by setting a higher maxSteps in Solve, but this can't be changed through GSParams at the moment.

**Comment by rmjarvis** [comment]
Thanks for the bug report.  I have reproduced this on my machine, so I'll look into what exactly is going on here.

**Comment by rmjarvis** [comment]
PR for fix is in #1042.

**Comment by rmjarvis** [comment]
Done #1042
